include/vsg/nodes/AbsoluteTransform.h

```diff
@@ -19,7 +19,7 @@ namespace vsg
 
     /// AbsoluteTransform is a transform node that provides a 4x4 matrix that is used to position subgraphs in absolute coordinate frame.
     /// During the RecordTraversal the matrix is directly pushed to the State::modelviewMatrixStack stack without the normal multiplication.
-    /// After the subgraph is traversed the matrix is popped from the State::modelviewMatrixStack.
+    /// After the subgraph is traversed the matrix is popped from the State::modelviewMatrixStack stack.
```

Comment:
Adding stack doesn't add extra information or clarity, just makes it awkward to read as the variable name is very descriptive and includes the word Stack to make clear what it is.

Reply:
The documentation for the MatrixTransform class does the same ('pushed to the State::modelviewMatrixStack stack.') so I thought I'd make it consistent (but I guess I made it consistent in the wrong manner)